content/docs/contributing/to-docs/blog-posts.md

```diff
@@ -6,95 +6,7 @@ toc: true
 weight: 600
 ---
 
-Blog posts are an important and semi-formal way to deliver announcements, insights, learnings, and professional opinions for the **Open 3D Engine (O3DE)** organization and the greater public. Part of growing as a community is establishing a strong blog presence, and we'd love for you to contribute! Here's how.
+# Blog Process
```

Comment:
I would move this heading after any introductory information on blogs and rename it to something like "Submit a blog" or "Blog submission process".  Headings should use sentence casing.